src/main.c

```diff
@@ -253,9 +257,20 @@ main_loop()
   pass_through_filter(TAG_OUTPUT,""); /* If something is wrong with filter, get the error NOW */
   set_echo(FALSE);		/* This will also put the terminal in CBREAK mode */
   test_main(); 
+
+  gettimeofday(&start_time, NULL);
```

Comment:
Consider using clock_gettime(CLOCK_MONOTONIC) instead gettimeofday. Using gettimeofday can result in erratic behavior if NTP decides to adjust the system clock, for example.

Reply:
Right, thanks !
I was tempted to use CLOCK_MONOTONIC_RAW,  but it's Linux specific